### PR TITLE
116: Snapcraft.yaml flow

### PR DESCRIPTION
**Pull request by katiefenn**
- Added "Learn the basics" link
- Added get started with template link, with template yaml file
- Added "Install it on your PC" link

### CODE REVIEW

File: mocks/service/github/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ router.post('/login/oauth/access_token', responses.okayAuthenticated);
 
 router.get(
   '/repos/:owner/:name/contents/snapcraft.yaml',
-  responses.okaySnapcraftYamlFound
+  responses.errorSnapcraftYamlNotFound
```
Comment (bartaz) [inline mocks/service/github/index.js]: We want that to be default mock response for now?
Probably not worth to make it conditional ATM depending on which mocked repo is sent in params...

File: mocks/service/launchpad/index.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ router.get('/devel/\\+snaps', (req, res) => {
       entries: [
         {
           resource_type_link: `${base}/#snap`,
-          git_repository_url: `${prefix}mock-repo-1`,
+          git_repository_url: `https://github.com/katiefenn/parker`,
```
Comment (bartaz) [inline mocks/service/launchpad/index.js]: Commited by mistake?

File: src/common/components/repository-row/index.js [hunk 3]
```diff
@@ -65,8 +70,21 @@ class RepositoryRow extends Component {
...
           <Data col="100">
-            This repo needs a snapcraft.yaml file, so that Snapcraft can make it buildable, installable, and runnable.
-            {/* TODO: add more info/links as in spec */}
+            <p>
+              This repo needs a snapcraft.yaml file,
+              so that Snapcraft can make it buildable,
+              installable, and runnable.
+            </p>
+            <p>
+              <a href={ LEARN_THE_BASICS_LINK } target="_blank">Learn the basics</a>,
```
Comment (bartaz) [inline src/common/components/repository-row/index.js]: We may need them with 'external link' icon, but we don't have this style globally yet (just in components that used that before...)

File: mocks/service/github/index.js (hunk 1 above)
Comment (katiefenn) [inline mocks/service/github/index.js]: @bartaz: Yeah, this crept in by accident.

File: mocks/service/launchpad/index.js (hunk 2 above)
Comment (katiefenn) [inline mocks/service/launchpad/index.js]: @bartaz Bingo 😞

**Pull request merged**